### ISSUE TITLE
Android-Java: Installing Java JDK

### ISSUE THREAD

**Issue by Rohit-2602** [girlscript/winter-of-contributing#34]
## Description ->
- What is Java JDK
- Give a brief on how to download and install java on a device (Mac, Windows, Linux).

## Source ->
- https://www.guru99.com/install-java.html

## Note ->
- Changes should be made inside the Android_Development/Java directory & Android_Development_with_java branch.
- Issue only for GWOC'21 contributors.
- Issue will be assigned on a first come first serve basis, 1 Issue == 1 PR.

Must-Follow: Contributing Guidelines & Code of Conduct before start Contributing.

**Comment by sumitmukharjeeeeee** [comment]
Can I take this one and Till when Can I submit it??

**Comment by Rohit-2602** [comment]
@sumitmukharjeeeeee yes sure go on. There is no such limit on time, but try to complete it as soon as possible.

**Comment by sumitmukharjeeeeee** [comment]
All right

**Comment by VikasPandey121** [comment]
Please keep these steps in mind while creating a pull request:

- Your Contributing file should be in this directory `Android_Development/Java/` with the file name starting from 1.1 for instance `1.1 Installing Java JDK`
Note:- If you have multiple files to commit then add a tag at the beginning of your file name like `1.1.1`, `1.1.2` etc
- Create a new branch and push your PR on `Android_Development_With_Java` branch.
- Pull request should contain only one commit
Note: If you have multiple commits then `squash` your commit.

**Comment by sumitmukharjeeeeee** [comment]
All right I am working on it

**Comment by sumitmukharjeeeeee** [comment]
Assign me again please I want to start over

**Comment by Rohit-2602** [comment]
@sumitmukharjeeeeee  can you please give a progress update about the issue and if you are facing any problem feel free to ask.

**Comment by issue31** [comment]
Will you please assign this issue to me?
I want to contribute to this issue via video format.

**Comment by VikasPandey121** [comment]
> 

Yes, sure @issue31. I have assigned this issue to you.
